### PR TITLE
Create design from scratch and added docker target in makefile 

### LINKED ISSUE #57: [Bug] Docker target missing in `Makefile`
#### Current Behavior

Documentation mentions `make docker` will make the docker image, but the docker target is missing from the `Makefile`.

#### Expected Behavior

Docker image gets built successfully.

#### Screenshots/Logs
Also after running `docker-compose build` and `docker-compose up` manually image is built with no error
logs
```
layer5io-docs-site-1 | hugo: downloading modules …
layer5io-docs-site-1 | hugo: collected modules in 29274 ms
layer5io-docs-site-1 | Start building sites … 
layer5io-docs-site-1 | hugo v0.111.3-5d4eb5154e1fed125ca8e9b5a0315c4180dab192+extended linux/amd64 BuildDate=2023-03-12T11:40:50Z VendorInfo=hugoguru
layer5io-docs-site-1 | 
layer5io-docs-site-1 |                    | EN  
layer5io-docs-site-1 | -------------------+-----
layer5io-docs-site-1 |   Pages            |  1  
layer5io-docs-site-1 |   Paginator pages  |  0  
layer5io-docs-site-1 |   Non-page files   |  5  
layer5io-docs-site-1 |   Static files     | 58  
layer5io-docs-site-1 |   Processed images |  0  
layer5io-docs-site-1 |   Aliases          |  0  
layer5io-docs-site-1 |   Sitemaps         |  0  
layer5io-docs-site-1 |   Cleaned          |  0  
layer5io-docs-site-1 | 
layer5io-docs-site-1 | Built in 5010 ms
layer5io-docs-site-1 | Watching for changes in /src/{assets,content,layouts,package.json,static}
layer5io-docs-site-1 | Watching for config changes in /src/hugo.toml, /src/go.mod
layer5io-docs-site-1 | Environment: "DEV"
layer5io-docs-site-1 | Serving pages from memory
layer5io-docs-site-1 | Running in Fast Render Mode. For full rebuilds on change: hugo server --disableFastRender
layer5io-docs-site-1 | Web Server is available at http://localhost:1313/ (bind address 0.0.0.0)
layer5io-docs-site-1 | Press Ctrl+C to stop
```

but visiting https://localhost:1313 shows `Page Not Found`

image

#### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by vaibhavpnimkar**
**Notes for Reviewers**

This PR fixes #57  and #77 

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by vaibhavpnimkar** [comment]
[Screencast from 25-11-23 11:50:11 PM IST.webm](https://github.com/layer5io/docs/assets/128807866/4d5f9d14-5a8b-43d1-b81b-c2af337b92fd)
`make docker ` command is working now

**Comment by abhijeetgauravm** [comment]
Hey @vaibhavpnimkar Let's discuss this on website call tomorrow at 6:30 PM IST / 7 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

File: content/en/meshmap/getting-started/starting-scratch.md [hunk 1]
```diff
@@ -2,15 +2,52 @@
...
+
+Upon entering MeshMap, navigate to the "Designer" mode by selecting the "Designer" tab from the main menu. This will open up the Designer toolbox, your gateway to creating intricate infrastructure designs.
+
+## Step 2: Giving Your Design a Name
+
+Initiate your design process by clicking on the "New Design" button in the Designer panel. Provide a meaningful name for your design, reflecting its purpose or key components.
+
+## Step 3: Browsing Available Components
+
+Explore the extensive toolbox in the Designer mode, offering a diverse array of components such as Kubernetes clusters, service mesh options, and cloud-native application elements. Each component is designed to seamlessly integrate into your infrastructure.
```
Comment (theBeginner86) [inline content/en/meshmap/getting-started/starting-scratch.md:24]: Please don't highlight service meshes as these are just a set kind of infrastructure that can be managed. Highlighting them sends a wrong message to users.

File: content/en/meshmap/getting-started/starting-scratch.md [hunk 2]
```diff
@@ -2,15 +2,52 @@
...
+
+Fine-tune your design by configuring each component using the Configuration panel. MeshMap's intuitive interface enables you to adjust settings and parameters effortlessly. Your design will automatically update with each change made.
+
+## Step 7: Design Actions and Interactions
+
+Explore additional design actions by right-clicking on components. Group components together or establish connections between them according to your needs. MeshMap's flexibility allows for a dynamic and responsive design experience.
+
+## Step 8: Merging Designs
+
+For enhanced collaboration and scalability, drag and drop existing designs into your canvas. This feature enables you to merge multiple designs seamlessly, creating a unified and comprehensive view of your infrastructure.
```
Comment (theBeginner86) [inline content/en/meshmap/getting-started/starting-scratch.md:44]: Please don't use abstract speech

File: Dockerfile [hunk 3]
```diff
@@ -1,4 +1,16 @@
-FROM klakegg/hugo:ext-alpine
+###############
+# Build Stage #
+###############
+FROM jakejarvis/hugo-extended:latest as builder
```
Comment (theBeginner86) [inline Dockerfile:4]: Thanks for this.
Is this a verified published for Hugo site?

Comment (theBeginner86) [reply Dockerfile:4]: Would you please consider using anyone of these Dockerfiles as examples?
https://github.com/google/docsy/blob/main/Dockerfile
https://github.com/google/docsy-example/blob/main/Dockerfile

File: content/en/meshmap/getting-started/starting-scratch.md
[the remark this answers is not in the document]
Comment (vaibhavpnimkar) [reply content/en/meshmap/getting-started/starting-scratch.md:1]: okay  noted !!!

[the remark this answers is not in the document]
Comment (vaibhavpnimkar) [reply content/en/meshmap/getting-started/starting-scratch.md:1]: should i  consider adding images for each action ?

File: Dockerfile (hunk 3 above)
Comment (vaibhavpnimkar) [reply Dockerfile:4]: okay will update jakejarvis/hugo-extended with docsy-example

File: content/en/meshmap/getting-started/starting-scratch.md
[the remark this answers is not in the document]
Comment (leecalcote) [reply content/en/meshmap/getting-started/starting-scratch.md:1]: Yes, if in doing so, you are describing the underlying system behavior. No, if you are simply giving the user a step-by-step series of instructions about how to click through the UI. If we have to provide them the step-by-step series of instructions on how to click through the UI, then we have failed our UX test, and I need to make UX changes.

[the remark this answers is not in the document]
Comment (vaibhavpnimkar) [reply content/en/meshmap/getting-started/starting-scratch.md:1]: okay i almost  got it  @leecalcote will update soon !!

File: Dockerfile (hunk 3 above)
Comment (leecalcote) [reply Dockerfile:4]: Good deal...

**Comment by vaibhavpnimkar** [comment]
@leecalcote @theBeginner86   updated jakejarvis/hugo-extended with docsy-example and also updated getting started docs

**Comment by abhijeetgauravm** [comment]
Hey @vaibhavpnimkar  Let's discuss this on website call tomorrow at 6:30 PM IST / 7 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

**Comment by leecalcote** [comment]
@vaibhavpnimkar thanks for contributing here. You've done some good work.

**Review by theBeginner86 — APPROVED** [review]
Thanks @vaibhavpnimkar 🚀

**Pull request merged**